pkgs/development/python-modules/pybind11/setup-hook.sh

```diff
@@ -0,0 +1,10 @@
+pybind11PythonConfig () {
+  cmakeFlagsArray+=(
```

Comment:
This is only for cmake and when invoked via nix. pybind11 is probably invoked differently. Not sure how to represent that the best.

Reply:
I think this is acceptable.